Sdk/Conversation/SwrveConversationStyler.m

```diff
@@ -79,8 +90,9 @@ +(void) styleButton:(SwrveConversationUIButton*)button withStyle:(NSDictionary*)
     }
     NSString *bgHexColor = [self colorFromStyle:[style objectForKey:kSwrveKeyBg] withDefault:kSwrveDefaultColorBg];
     UIColor *bgUIColor = [self convertToUIColor:bgHexColor];
+    float borderRadius = [self convertBorderRadius:[[style objectForKey:kSwrveKeyBorderRadius] floatValue]];
```

Comment:
don't all primitive types get defaulted to 0? so it'll give us 0 anyway. 
I'll add a check just in case